.changes/unreleased/FEATURES-20231102-160132.yaml

```
@@ -0,0 +1,5 @@
+kind: FEATURES
+body: 'types/basetypes: Added TupleType and TupleValue implementations'
```

Nit: I wonder if we should potentially call this out as an enhancement rather than a feature and describe it a little more, since ultimately we don't anticipate provider developers to interact with this directly too much.

```suggestion
kind: ENHANCEMENTS
body: 'types/basetypes: Added `TupleType` and `TupleValue` implementations, which are only necessary for dynamic value handling'
```

---

Yeah this makes sense, I was between the two 😆 - ⚡